Formula/node_exporter.rb

```diff
@@ -14,13 +14,30 @@ class NodeExporter < Formula
 
   depends_on "go" => :build
 
+  resource "promu" do
+    # check and update upon next release
+    # (matches <https://github.com/prometheus/node_exporter/blob/v0.17.0/Makefile.common#L68-L69>)
+    url "https://github.com/prometheus/promu/releases/download/v0.2.0/promu-0.2.0.darwin-amd64.tar.gz"
```

Comment:
Let's not download binaries **at all**, please.

Reply:
Do you mean that the formula should build the `promu` utility from source, or did you have something else in mind? Regardless, I'm happy to oblige!

Reply:
Yes, let's build from source.